_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
+    * Go to your **Applications/Utilites** folder. Double-click **Terminal**.
+* Once the **Terminal** is open, you will need to navigate to the folder in which you put all the files. As an example, let's say you put them on the Desktop. You would type the following in your **Terminal** and hit "return":
+
+```    
+cd ~/Desktop
+```
+
+cd stands for change directory. Once you hit "return", you will be working inside the Desktop. For another example, let's say you put your files in a folder called KittyCADPractice within the Documents folder. This would look like so:
+
+```
+cd ~/Documents/KittyCADPractice
+```
+
+You will now be working inside the folder you created called KittyCADPractice.
+* Once inside the correct folder, input the following command into **Terminal**. (Note: if you are signed in and have made an API token, we auto-populated your token below)
+
+<Token>
+    {(token, platform) =>
+        [
+            platform === 'windows'
+                ? `$env:KITTYCAD_API_TOKEN = '${token}'`
+                : `export KITTYCAD_API_TOKEN="${token}"`,
+            'pip3 install virtualenv',
+            'python3 -m venv venv',
+            platform === 'windows'
+                ? 'venv/Scripts/Activate.ps1'
+                : 'source venv/bin/activate',
+            'pip3 install -r requirements.txt',
+            'python3 convert.py',
+        ].join('\n')
+    }
+</Token>
+
+As long as everything went smoothly, you will see the following output:
```

Comment:
You could also mention that they should see `output.stl` in that folder now as well.